### ISSUE TITLE
nightly: cargo cov creates empty report

### ISSUE THREAD

**Issue by shahn** [kennytm/cov#18]
Pretty much as the title says, on nightly cargo cov doesn't create any gcda or gcno files and the browser shows an empty html page

**Comment by king6cong** [comment]
The same here

**Comment by mathstuf** [comment]
I'm also seeing this with 1.24.0 final.

**Comment by lo48576** [comment]
Same for me.

```
$ rustc +nightly --version
rustc 1.26.0-nightly (80785a547 2018-03-30)
$ cargo +nightly --version
cargo 1.26.0-nightly (d63299b6e 2018-03-28)
$
```

When running `cargo +nightly cov test` normally, it generates no `*.gdca` files.
I tried
```
RUSTFLAGS='-Z profile -C codegen-units=1' CARGO_CFG_REGEX_DISABLE_AUTO_OPTIMIZATIONS=1 cargo +nightly cov test --lib
```
and it generates `*.gdca` files, but it emits many
```
profiling: (path).gdca: cannot merge previous GCDA file: corrupt arc tag (0xfoobar)
```
error messages for my crate and exit unsuccessfully...

**Comment by lo48576** [comment]
```
RUSTFLAGS='-Z profile -C codegen-units=1' CARGO_CFG_REGEX_DISABLE_AUTO_OPTIMIZATIONS=1 cargo +nightly cov build --lib
```
This successfully exited, and I ran `cargo +nightly cov report --open`, then it generates html report with no entries... :roll_eyes:

**Comment by mssun** [comment]
Any updates?

**Comment by themasch** [comment]
I'm also having this issue on 1.25.0. `cargo cov test` runs, executes all tests and creates no files. Report is empty, too. 

But since others seem to have this issue on nightly only, maybe I am doing something wrong?

**Comment by kaiba696** [comment]
^ +1

**Comment by kennytm** [comment]
Fixed by 6ecd86a3c273349eaf2734b304d52c53910136eb and 05d1c1ec6ab39468f009ec3e7d1165791da3b628.
